playbooks/roles/edxapp/files/microsite_theming.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All Rights Reserved.
+
+# Remove if edx-microsite folder exists
+if [[ -d /edx/app/edxapp/edx-microsite ]]; then
```

Comment:
i just followed the pattern we used before on the same file below for kitchen sink course.